src/lv_misc/lv_style.c

```diff
@@ -90,13 +90,20 @@ bool lv_style_remove_prop(lv_style_t * style, lv_style_prop_t prop)
         if(props[i] == prop) {
             lv_style_value_t * values = (lv_style_value_t *)style->v_p.values_and_props;
```

Comment:
It's not related to your changes but it seems  `old_value` would be a better name for this.

Reply:
Done.